tests/unit/test_dict_merger.py

```diff
@@ -87,6 +87,18 @@ def test_merge_with_nothing():
     m.merge()
     assert m.merged_root == {'some': 'other object'}
 
+    m = SkipListsMerger({'some': 'other object'},
+                        NOTHING, {'some': 'other object'},
+                        DictMergerOps.FALLBACK_KEEP_HEAD)
+    m.merge()
+    assert m.merged_root == NOTHING
+
+    m = SkipListsMerger({'some': 'other object'},
+                        {'some': 'other object'}, NOTHING,
+                        DictMergerOps.FALLBACK_KEEP_HEAD)
+    m.merge()
+    assert m.merged_root == NOTHING
```

Comment:
are you 100% sure that `NOTHING` can't be present in the output? I believe it's produced only by the unifier when elements are missing in some of the versions, but it would be good to double check.

Reply:
Yes, we get instead of omitted value something like `[NOTHING, 2]` instead `[2]` as a result of a merge. It is also causing some issues like attribute errors in inspire-json-merger